### PR TITLE
[TW-1023] Doc sunset of deprecated GitHub and AWS API Gateway integrations

### PR DESCRIPTION
**Pull request by mark-anthony-postman**

### CODE REVIEW

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM!

**Review by petewitcher-postman — APPROVED** [review]
LGTM!

**Review by shashankawasthi88 — APPROVED** [review]
Looks good

**Review by jkonrath-postman — APPROVED** [review]
LGTM

File: src/pages/docs/developer/postman-api/intro-api.md [hunk 1]
```diff
@@ -50,3 +50,9 @@ You can get started with the API by [forking the collection](https://www.postman
 You will need an [API key](/docs/developer/postman-api/authentication/#generate-a-postman-api-key) to access the Postman API.
 
 > The Postman API is [rate limited](/docs/developer/postman-api/postman-api-rate-limits/).
+
+## About v9 and v10 APIs
+
+The format for APIs created in Postman changed in Postman v10. The Postman API now supports endpoints for working with APIs created in v10. To use these endpoints, you must pass an `Accept` header with the value `application/vnd.api.v10+json`. Learn more about [using the Postman v10 endpoints for APIs](https://www.postman.com/postman/workspace/postman-public-workspace/folder/12959542-daed38dc-2fd4-4373-a73b-cbdbb72f5819).
```
Comment (mark-anthony-postman) [inline src/pages/docs/developer/postman-api/intro-api.md:56]: @insomniac12 Do you know what the final link will be for linking to the new "v10 API" folder in the public workspace? I'd like to include it here.

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Review by shashankawasthi88 — APPROVED** [review]
Looks good to me

**Pull request merged**